src/onegov/feriennet/forms/attendee.py

```diff
@@ -318,7 +318,7 @@ def populate_tos(self) -> None:
 
         layout = DefaultLayout(self.model, self.request)
 
-        self.accept_tos.description = Markup(render_macro(  # noqa: MS001
+        self.accept_tos.description = Markup(render_macro(
```

Comment:
This one should still trigger, so if it doesn't, that's a bug in the ruff implementation I'll have to look into.

Reply:
It didn't trigger!

Reply:
I see the problem, the Ruff implementation is correct, this is using `chameleon.utils.Markup`, we should be using `markupsafe.Markup` here too. Technically they're interchangeable as far as template rendering is concerned, but it's better to be consistent.

We can also add the following section to `pyproject.toml`, so this other `Markup` class is detected as well:

```toml
[ruff.lint.ruff]
extend-markup-names = ["chameleon.utils.Markup"]
```

Reply:
Added it if somebody ever uses it